pkg/ccl/changefeedccl/changefeed.go

```diff
@@ -92,3 +94,14 @@ func initialScanFromOptions(opts map[string]string) bool {
 	_, noInitialScan := opts[changefeedbase.OptNoInitialScan]
 	return (cursor && initialScan) || (!cursor && !noInitialScan)
 }
+
+func setJobRunningStatus(ctx context.Context, j *jobs.Job, fmtOrMsg string, args ...interface{}) {
+	status := jobs.RunningStatus(fmt.Sprintf(fmtOrMsg, args...))
+	if err := j.RunningStatus(ctx, nil,
```

Comment:
This API makes me very sad. Can we pull it under the Update method on the job so that we can update the running status transactionally with the high watermark?